docs/en/Running-Alluxio-on-Mesos.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: global
+title: Running Alluxio on Mesos
+nickname: Alluxio on Mesos
+group: Deploying Alluxio
+priority: 4
+---
+
+Alluxio can be deployed through Mesos. This allows Mesos to manage the resources used by Alluxio.
```

Comment:
You may want to clarify this. Ie. does it manage JVMs only or also ramdisks, etc.